homeworks/09-node-chokidar/.gitignore

```diff
@@ -1,4 +1,4 @@
 node_modules
 .idea
 bin
-
+data
```

Comment:
If `data` folder doesn't exist in project then the application will throw unhandled errors